templates/unicorn.rb.j2

```diff
@@ -0,0 +1,134 @@
+# Sample verbose configuration file for Unicorn (not Rack)
```

Comment:
`ansible_managed`
Remove these comment blocks, just comment any non-obvious option.
I will rereview after that.

Reply:
done